frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx

```diff
@@ -0,0 +1,407 @@
+        inactive: 16,
+    },
+    {
+        date: '2023-09-30',
+        total: 186,
+        active: 169,
+        inactive: 17,
+    },
+    {
+        date: '2023-10-07',
+        total: 188,
+        active: 173,
+        inactive: 15,
+    },
+    {
+        date: '2023-10-14',
+        total: 181,
+        active: 166,
+        inactive: 15,
+    },
+    {
+        date: '2023-10-21',
+        total: 192,
+        active: 177,
+        inactive: 15,
+    },
+    {
+        date: '2023-10-28',
+        total: 183,
+        active: 164,
+        inactive: 19,
+    },
+    {
+        date: '2023-11-04',
+        total: 200,
+        active: 180,
+        inactive: 20,
+    },
+    {
+        date: '2023-11-11',
+        total: 212,
+        active: 189,
+        inactive: 23,
+    },
+    {
+        date: '2023-11-18',
+        total: 204,
+        active: 177,
+        inactive: 27,
+    },
+    {
+        date: '2023-11-25',
+        total: 200,
+        active: 173,
+        inactive: 27,
+    },
+    {
+        date: '2023-12-02',
+        total: 200,
+        active: 175,
+        inactive: 25,
+    },
+    {
+        date: '2023-12-09',
+        total: 200,
+        active: 176,
+        inactive: 24,
+    },
+    {
+        date: '2023-12-16',
+        total: 215,
+        active: 186,
+        inactive: 29,
+    },
+    {
+        date: '2023-12-23',
+        total: 221,
+        active: 195,
+        inactive: 26,
+    },
+    {
+        date: '2023-12-30',
+        total: 214,
+        active: 184,
+        inactive: 30,
+    },
+    {
+        date: '2024-01-06',
+        total: 204,
+        active: 173,
+        inactive: 31,
+    },
+    {
+        date: '2024-01-13',
+        total: 215,
+        active: 181,
+        inactive: 34,
+    },
+];
+
+const createData = (theme: Theme) => ({
+    labels: mockData.map((item) => item.date),
+    datasets: [
+        {
+            label: 'Total users',
+            data: mockData.map((item) => item.total),
+            borderColor: theme.palette.primary.main,
+            backgroundColor: theme.palette.primary.main,
+            fill: true,
+        },
+        {
+            label: 'Inactive users',
+            data: mockData.map((item) => item.inactive),
+            borderColor: theme.palette.error.main,
+            backgroundColor: theme.palette.error.main,
+            fill: true,
+        },
+        {
+            label: 'Active users',
+            data: mockData.map((item) => item.active),
+            borderColor: theme.palette.success.main,
+            backgroundColor: theme.palette.success.main,
+            fill: true,
+        },
+    ],
+});
+
+const createOptions = (theme: Theme, locationSettings: ILocationSettings) =>
+    ({
+        responsive: true,
+        plugins: {
+            legend: {
+                position: 'bottom',
+            },
+            tooltip: {
+                callbacks: {
+                    title: (tooltipItems: any) => {
```

Comment:
any type

Reply:
I'd either use default type TooltipItem as other places or pass unknown to a parser that produces a correct type